org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/publisher/Publisher.java

```diff
@@ -1386,6 +1375,33 @@ else if (p.getValue().equals("no-broken-links"))
         else if (p.getValue().equals("show-reference-messages"))
           showReferenceMessages = true;
         
+      } else if (p.getCode().equals("language") && p.hasValue()) {
+        final String language = p.getValue();
+        if(language != null && language.length() > 0) {
+          if(this.translator == null) {
+            this.translator = new TranslationImplementation();
+          }
+          this.translator.setLang(language);
+        }
+      } else if (p.getCode().equals("languages") && p.hasValue()) {
+        System.out.println("Languages parameter found. No support yet.");
+        /*
+        if(this.translator == null) {
+            this.translator = new TranslationImplementation();
+          }
+        final String languages = p.getValue();
+        if(languages != null && languages.length() > 0) {
+          final String firstLanguage = Arrays.stream(languages.split(","))
+                  .map(String::trim)
+                  .filter(s -> s.length() > 0)
+                  .findFirst()
+                  .orElseGet(() -> {
+                    System.out.println("Language parameter defined, but no valid languages found");
+                    return null;
+                  });
+          ((TranslationImplementation)this.translator).setLang(firstLanguage);
+        }
+        */
```

Comment:
I"m actively trying to reduce the amount of commented out code in the libraries right now. If a feature isn't ready to be checked in, let's try and keep it in a branch. Using the master branch to store wip code really isn't ideal.